src/components/xl-newsletter/xl-newsletter.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { StaticImage } from "gatsby-plugin-image";
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { Input } from "../input";
+import { Button } from "../button";
+import { Close } from "../icons"
+
+import { classnames } from "../../helpers";
+
+import * as styles from "./xl-newsletter.module.scss";
+
+const XlNewsletter = ({ onClick }) => {
```

Comment:
Why do you pass the handle action as prop? I think XlNewsletter should handle the onClick action

Reply:
I don't have the action, but, you have the reason